src/Divante/MagentoIntegrationBundle/Application/Product/MappedProductService.php

```diff
@@ -46,6 +46,7 @@ public function getProducts(string $ids, string $instanceUrl, string $storeViewI
             ];
         }
         $configuration = reset($configurations);
+        AbstractObject::setGetInheritedValues(true);
```

Comment:
I think it should be added in integratedObjectRepository function `getObjects`, so in future implementation we will no have to think about it when fetching objects

Reply:
It doesn't work as it should, wen we put this into getObjects method.